.github/workflows/manubot.yaml

```diff
@@ -24,6 +24,13 @@ jobs:
         with:
           # fetch entire commit history to support get_rootstock_commit
           fetch-depth: 0
+      - name: Cache
+        uses: actions/cache@v1
+        with:
+          path: ci/cache
+          key: ci-cache-${{ github.ref }}
+          restore-keys: |
+            ci-cache-refs/heads/master
```

Comment:
the point of these three lines is to make it so that PRs can never write to the master branch cache but can still read from it if they don't have their own cache yet